src/scheduler/parse.c

```diff
@@ -103,7 +103,7 @@ parse_config(char *fname)
 	FILE *fp;			/* file pointer to config file */
 	char *buf = NULL;
 	int buf_size = 0;
-	char buf2[8192];		/* general purpose buffer */
+	char buf2[PBS_FGETS_LINE_LEN];	/* general purpose buffer */
```

Comment:
Why PBS_FGETS_LINE_LEN?  This has nothing to do with fgets.

Reply:
You are right, this buffer isn't used with fgets. Reverting it.